stable/metrics-server/values.yaml

```diff
@@ -90,3 +90,7 @@ service:
   annotations: {}
   port: 443
   type: ClusterIP
+
+# The values to set in the PodDisruptionBudget spec (minAvailable/maxUnavailable)
+# If not set then a PodDisruptionBudget will not be created
+podDisruptionBudget:
```

Comment:
Could you add a (commented out) example of what is users should add here? Right now, one has to look at the template.

Reply:
Will do.  I have another nearly-identical PR pending for the kubernetes-dashboard chart.  Do you advise preemptively making the same changes there?

Reply:
To clarify:  all the changes, not just the additional documentation in values.yaml.

Reply:
I would advice that yes. The kubernetes-dashboard maintainer may have other preferences though.

Reply:
I went ahead and made the changes over there as well.  No response yet, but I'm happy to do it whichever way they prefer.